nextNthLetter.py

```diff
@@ -0,0 +1,24 @@
+# Write a procedure, shift_n_letters which takes as its input a lowercase
+# letter, a-z, and an integer n, and returns the letter n steps in the
+# alphabet after it. Note that 'a' follows 'z', and that n can be positive,
+#negative or zero.
+import string
+characters = string.ascii_lowercase
+strings = []
+for i in range(len(characters)):
+    strings.append(characters[i])
+
+def shift_n_letters(letter, n):
+    # Your code here
+    i = strings.index(letter)
```

Comment:
There is nothing wrong in your function. But I would like you to know about another method using ASCII values. You can find the index directly from ascii.